portfolio/src/main/webapp/projects.css

```diff
@@ -0,0 +1,69 @@
+.font {
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.holder {
+    margin-bottom: 50px;
+}
+
+.navbar-link {
+    color: white;
+} :hover {
+    color: black;
+}
+
+.title {
+  margin-top: 50px;
+  font-weight: 600;
+  font-size: 48px;
+}
+
+.tag-line {
+  font-size: 24px;
+  margin-bottom: 15px;
+  font-style: italic;
+}
+
+.proj-title {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  font-weight: 500;
+  font-size: 24px;
+}
+
+.flex {
+  margin-top: 35px;
+}
+
+.proj-description {
+  margin-left: 2%;
```

Comment:
Do you want the margins to expand and contract with the container size? Try out how this looks with very large and very small screen widths in particular. It might be better to use absolute values with responsive break-points.

Reply:
replaced with better use of bootstrap cols